kolibri_explore_plugin/assets/src/routes/index.js

```diff
@@ -42,7 +42,7 @@ export default [
     name: PageNames.TOPICS_TEST,
     path: '/topics/:channel_id/test',
     handler: toRoute => {
-      store.commit('topicsTree/SET_CUSTOM_APP_CONTENT', { test: true });
+      store.commit('topicsTree/SET_CUSTOM_APP_PARAMETERS', { test: 'true' });
```

Comment:
I think that this change to string is not needed, the encode function will translate this to string.

Reply:
You are right, the encode will do it.